lib/jdomodels/src/main/resources/sql/GetEntityAccessRestrictions.sql

```diff
@@ -24,14 +24,23 @@ WITH
 		SELECT EAR.*, if(AA.STATE = 'APPROVED', TRUE, FALSE) AS APPROVED FROM EAR
 			LEFT JOIN ACCESS_APPROVAL AA
 				ON (EAR.REQUIREMENT_ID = AA.REQUIREMENT_ID AND AA.ACCESSOR_ID = :userId AND AA.STATE = 'APPROVED')
-	 )
+	 ),
+    ACC AS(
+        SELECT EAR.REQUIREMENT_ID AS REQUIREMENT_ID, IF(RAT.STRING_ELE ='EXEMPTION_ELIGIBLE', TRUE, FALSE) AS IS_EXAMPTION_ELIGIBLE FROM EAR
+            LEFT JOIN ACL ON (EAR.REQUIREMENT_ID= ACL.OWNER_ID)
+            LEFT JOIN ACL_RESOURCE_ACCESS ARA ON (ACL.ID = ARA.OWNER_ID)
+            LEFT JOIN ACL_RESOURCE_ACCESS_TYPE RAT ON (ARA.ID = RAT.ID_OID)
+            WHERE  ARA.GROUP_ID IN (:usersGroups) AND RAT.STRING_ELE ='EXEMPTION_ELIGIBLE'
```

Comment:
no where and add each condition to is corresponding on clause.
LEFT JOIN ACL_RESOURCE_ACCESS_TYPE RAT ON (ARA.ID = RAT.ID_OID AND ARA.GROUP_ID IN (:usersGroups) AND RAT.STRING_ELE ='EXEMPTION_ELIGIBLE')